src/core/layout/navbar/header/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/128524824-2c5eff72-5c3a-4af0-b35c-fe0ca75dcdbf.png)
This component/folder became unused. Let's remove it.

Reply:
Done